back-end/helpers/db-helpers.js

```diff
@@ -24,7 +20,38 @@ async function getRowFromDb(queryString) {
   return results.rows[0];
 };
 
+async function getUserByEmail(email) {
+  const query = `SELECT password, id FROM users WHERE email = '${email}'`;
+  const userData = await getRowFromDb(query);
+  return userData;
+}
+
+async function getUserGitHubInfo(userID) {
+  const userGitHubInfoQuery = `SELECT * FROM github_info WHERE user_id=${userID}`;
+  const userGitHubData = await getRowFromDb(userGitHubInfoQuery);
+  return userGitHubData;
+}
+
+async function getFullUserProfile(userID) {
```

Comment:
Consolidate information from the `/users` table `github_info` table into a single users representation.